include/SFML/Window/Event.hpp

```diff
@@ -100,7 +100,7 @@ class Event
     };
 
     ////////////////////////////////////////////////////////////
-    /// \brief Mouse wheel events parameters (MouseWheelMoved)
+    /// \brief Mouse wheel events parameters (MouseWheelMoved) (deprecated)
```

Comment:
But... there is no paragraph to mark with it? There is only the \brief description line.